docs/Quickstart.md

```diff
@@ -0,0 +1,295 @@
+## Quickstart
```

Comment:
Think we still need this? Having docs in two places means we have a high chance of having them out of sync.